src/cascadia/TerminalSettingsModel/TerminalSettings.idl

```diff
@@ -26,6 +26,8 @@ namespace Microsoft.Terminal.Settings.Model
     {
         TerminalSettings();
 
+        Windows.Foundation.Collections.StringMap EnvironmentVariables;

```

Comment:
same note as prior about observability, but for TerminalSettings instead of Profile